tests/test_operator.py

```diff
@@ -467,26 +467,26 @@ def test_override_timefunction_data(self):
 
         # Run with default value
         a.data[:] = 1.
-        op(t=2)
-        assert (a.data[:] == 4.).all()
+        op(t=1)
+        assert (a.data[0] == 4.).all()
```

Comment:
Maybe would add
`assert (a.data[1] == 3.).all()`
as only `t=0` should be updated

Reply:
👍